### PR TITLE
Added distance calculation support for proto

### PR DESCRIPTION
**Pull request by arnasbr**
TimeFilterFastProto now supports distances calculation, in addition to travel_times

### CODE REVIEW

File: traveltimepy/dto/common.py [hunk 1]
```diff
@@ -107,6 +107,10 @@ class Property(str, Enum):
     FARES = "fares"
 
 
+class PropertyProto(int, Enum):
```
Comment (danielnaumau) [inline traveltimepy/dto/common.py:110]: you can add ferries too

**Review by danielnaumau — APPROVED** [review]

File: traveltimepy/dto/common.py (hunk 1 above)
Comment (arnasbr) [reply traveltimepy/dto/common.py:110]: I thought adding fares would be misleading for the user since we don't support it here for now. I can add it tho

**Comment by arnasbr** [comment]
@Donatas-L @pawel-wroniszewski @PKJonas can I get a quick review?

File: README.md [hunk 2]
```diff
@@ -342,17 +342,19 @@ larger limits on the amount of destination points.
 * country: ProtoCountry - Return the results that are within the specified country.
 * one_to_many: boolean - if one_to_many is equal to true, then it'll be a forward search (one to many matrix), false -
   backward search (many to one matrix). Default value is True.
+* properties: List[PropertyProto] - specifies which extra properties should be calculated in the response. 
 
 #### Returns:
 
-* travel_times: List[int] - The responses are in the form of a list where each position denotes either a travel time (in
-  seconds) of a journey, or if negative that the journey from the origin to the destination point is impossible.
+* results: TimeFilterProtoResponse - The response contains 2 lists, firstly travel times, where each position denotes either a travel time (in
```
Comment (Donatas-L) [inline README.md:349]: Probably contains 1 or 2 lists?

I would write it like this:

results: TimeFilterProtoResponse - The response contains:
* list of travel times, where each position denotes either a travel time (in seconds) of a journey, or if travel time is negative, that the journey from the origin to the destination point is impossible.
* (optional) list of distances where each position denotes distance (in meters) to the specified location.

Comment (arnasbr) [reply README.md:349]: Thanks, I was struggling to word this properly. Implemented.

**Review by Donatas-L — APPROVED** [review]

**Pull request merged**